Network-Security/Integration/aws-python-amazon-inspector-integration/c1ns-inspector-findings-template.yaml

```diff
@@ -0,0 +1,951 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  AwsRegion:
+    Type: String
+    Description: Please enter the region in which the lambda will be executed
+    Default : us-east-1
+    AllowedValues:
```

Comment:
Not a big issue, but can this be a variable to capture the region its deployed in over a map and parameter? like ${AWS::Region}

Reply:
Is that work when you deploy via cli as well or only from the UI?

Reply:
Will work in CLI. Also CLI can be configured for a default region. Its an AWS global variable and should populate accordingly